diff --git a/src/plugins/betterRoleContext/index.tsx b/src/plugins/betterRoleContext/index.tsx
index 9b5653a..ca42716 100644
--- a/src/plugins/betterRoleContext/index.tsx
+++ b/src/plugins/betterRoleContext/index.tsx
@@ -214,6 +214,13 @@ export default definePlugin({
                 match: /(?<=#{intl::CHANNEL_MEMBERS_A11Y_LABEL}.{0,200}?"aria-hidden":!0,)children:.{0,200}?(?:—|\\u2014) ",\i\]\}\)\]/,
                 replace: "onContextMenu:e=>$self.openRoleContextMenu(e,arguments[0]),$&"
             }
+        },
+        {
+            find: '["\\xa0\\u2014 ",',
+            replacement: {
+                match: /#{intl::CHANNEL_MEMBERS_A11Y_LABEL}.{0,150}?"aria-hidden":!0,/,
+                replace: "$&onContextMenu:e=>$self.openRoleContextMenu(e,arguments[0]),"
+            }
         }
     ],
 
