diff --git a/src/plugins/_api/badges/index.tsx b/src/plugins/_api/badges/index.tsx
index d91b4eae..2f192a8c 100644
--- a/src/plugins/_api/badges/index.tsx
+++ b/src/plugins/_api/badges/index.tsx
@@ -51,8 +51,7 @@ async function loadBadges(noCache = false) {
     if (noCache)
         init.cache = "no-cache";
 
-    DonorBadges = await fetch("https://badges.vencord.dev/badges.json", init)
-        .then(r => r.json());
+    DonorBadges = {};
 }
 
 let intervalId: any;
